### DISCUSSION TITLE
geopy

### DISCUSSION THREAD

**Discussion by clintonclinton2** [geopy/geopy#518] [Help]
como faca para pegar a minha localizcao actual e medir a distantancia em relacao outros pontos?
imagina que estou em casa e preciso saber qual eh o hotel mais proximo em kilometro apartir de uma aplicao, tendo em conta que as coordenadas de hoteis ja foram cdastradas no sistema, mas devem calcular  a distancia com base o usuario que esta  a usar aplicacao

**Comment by KostyaEsmukov** [comment]
Some geocoding services allow to use biases and viewports towards the given point, this might come helpful for this task.
